at_client/README.md

```diff
@@ -1,8 +1,104 @@
+- Cryptographic control of data access through personal data stores
+- No application backend needed
+- End to end encryption where only the data owner has the keys
+- Private and surveillance free connectivity
+
+We call giving people control of access to their data "*flipping the internet*".
+
+## Get Started
+
+Initially to get a basic overview of the SDK, you must read the [atsign docs](https://atsign.dev/docs/overview/).
+
+> To use this package you must be having a basic setup, Follow here to [get started](https://atsign.dev/docs/get-started/setup-your-env/).
+
+Check how to use this package in the [at_client installation tab](https://pub.dev/packages/at_client/install).
+
+## Usage
+
+- Set `AtClientPreferences` to your preferred settings.
+
+```dart
+Directory appSupportDir = await getApplicationSupportDirectory();
+AtClientPreference preferences = AtClientPreference()
+        ..rootDomain = 'root.atsign.org'
+        ..namespace = '.my_namespace'
+        ..hiveStoragePath = appSupportDir.path
+        ..commitLogPath = appSupportDirdir.path
+        ..isLocalStoreRequired = true;
+```
+
+- These `preferences` are used for your application.
+
+```dart
+AtClientManager atClientManagerInstance = await AtClientManager.getInstance().setCurrentAtSign(atSign, AtEnv.appNamespace, preferences);
+```
+
+- Update the user data using the `put()` method.
+
+```dart
+AtKey atKey = AtKey()
+        ..key = 'phone'
+        ..namespace = '.myApp';
+await atClientInstance.put(atKey, '+00 123-456-7890');
+```
+
+- Get the data using the `get()` method.
+
+```dart
+AtKey atKey = AtKey()
+        ..key='phone'
+        ..namespace = '.myApp';
+AtValue value = await atClientInstance.get(atKey);
+print(value.value); // +00 123-456-7890
+```
+
+- Delete the data using the `delete()` method.
+
+```dart
+bool isDeleted = await atClientInstance.delete(atKey);
+print(isDeleted); // true if deleted
+```
+
+- Sync the data to the server using the `sync()` method if needed.
+
+```dart
+late SyncService _syncService;
+_syncService = atClientManagerInstance.syncService;
+_syncService.sync(onDone: _onSuccessCallback); // prints 'Sync done' on done.
+
+void _onSuccessCallback() {
+  print('Sync done');
+}
+```
+
+- Notify the server that the data has changed using the `notify()` method.
+
+```dart
+AtClientManager atClientManagerInstance = AtClientManager.getInstance();
+MetaData metaData = Metadata()..ttl='60000'
+               ..ttb='30000'
+AtKey key = AtKey()..key='phone'
+          ..sharedWith='@bob'
+          ..metadata=metaData
+        ..namespace = '.myApp';
+String value = (await atClientInstance.get(atKey)).value;
+OperationEnum operation = OperationEnum.update;
+bool isNotified = await atClientManagerInstance.notify(atKey, value, operation);
+print(isNotified); // true if notified
+```
+
+AtClient has many more methods that are exposed. Please refer to the [atsign docs](https://atsign.dev/docs/overview/) for more details.
```

Comment:
this is just a demo link, this will be replaced with the other link on release time